RELEASES.md

```diff
@@ -1,3 +1,130 @@
+Version 1.54.0 (2021-06-29)
+============================
+
+Language
+-----------------------
+
+- [You can now use macros for values in key-value attributes.][83366]
+  While a seemingly minor addition on its own, this enables a lot of
+  powerful functionality when combined correctly. Most notably you can
+  now include external documentation in your crate by writing the following.
+  ```rust
+  #![doc = include_str!("README.md")]
+  ```
+
+- [You can now cast between unsized slice types (and types which contain
+  unsized slices) in `const fn`.][85078]
+- [TODO: stabilize member constraints][84701]
```

Comment:
cc @nikomatsakis or someone smarter to me :). What does stabilising member constraints mean for users? It was a bit hard to grasp the net effect of the feature from reading the issues.

Reply:
First pass:

You can now use multiple generic lifetimes in return position `impl Trait`, including on `async fn`

Reply:
Well, more so, you can now use multiple generic lifetimes in `impl Trait` *where the lifetimes don't explicitly outlive another*.

For example, you can now do
```rust
fn foo<'a, 'b>(...) -> impl Trait<'a, 'b> { .. }
```
whereas before you had to do (for example)
```rust
fn foo<'a, 'b: 'a>(...) -> impl Trait<'a, 'b> { .. }
```

Note that this is has already been conditionally stable in the desugared `impl Trait` of `async fn`, but can now is stabilized for all functions.